tests/e2e/common.go

```diff
@@ -122,11 +153,9 @@ func (td *OsmTestData) AreRegistryCredsPresent() bool {
 func (td *OsmTestData) InitTestData(t GinkgoTInterface) error {
 	td.T = t
 
-	if len(td.ctrRegistryServer) == 0 {
-		td.T.Errorf("Did not read any container registry (is CTR_REGISTRY set?)")
-	}
+	Expect(verifyValidInstallType(td.instType)).To(Succeed())
```

Comment:
Should we just return an error here instead if this fails so we can tell which test actually failed from the ginkgo back trace?

Reply:
y... yes, not sure why did I do that

Reply:
also fixed the `UpdateOSMConfig`